src/types/index.ts

```diff
@@ -0,0 +1,7 @@
+import { RunningJobsTypes, PendingJobsTypes, JobsHistoryTypes } from "./Job";
+
+export {
+    PendingJobsTypes,
```

Comment:
Job.ts file does not contain the imported job types.

Export the other types, Enumeration, TemporalExpression and Status

Reply:
done sir!